test/system_test.py

```diff
@@ -617,6 +597,7 @@ def test_install_default(self):
 
 
 @pytest.mark.no_tsa  # Irrelevant
+@pytest.mark.no_daint  # problem with gt4py and spack v21.1
```

Comment:
@abishekg7 @samkellerhals This means no more Daint tests for gt4py. Is this okay?

Reply:
Shall we remove the broken versions from gt4py?
I'd rather not have any versions around that we know are broken.